applications/launchpad_v2/src/App.tsx

```diff
@@ -18,26 +19,28 @@ function App() {
 
   return (
     <ThemeProvider theme={{}}>
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
+      <GlobalStyle />
+      <>
```

Comment:
you're wrapping single component inside a fragment, either GlobalStyle should be inside here to satisfy ThemeProvider children props or the fragment is not needed

Reply:
Fixed, my error.